wiki/How-to-write-tests.md

```diff
@@ -64,16 +64,16 @@ You can also use [this short Jest cheatsheet](https://devhints.io/jest)
 [RTL Cheatsheet](https://raw.githubusercontent.com/testing-library/react-testing-library/master/other/cheat-sheet.pdf)
 
 #### Which testing tool to use
-* For everything that is related to rendering components and therefore requires evaluations against the DOM
+* For everything related to rendering components and therefore requires evaluations against the DOM
 * If you are only testing functions and methods, please use plain Jest
 
 ### What to test?
-#### Do 
+#### Do
 * Utility methods
 * Complex implementations (eg: algorithms)
 * Anything that has edge cases
 * Core business logic
-* Services that are high-risk
+* high-risk Services
```

Comment:
Same here again. For each bullet point the first letter should be a capital letter or not